.github/workflows/release.yml

```diff
@@ -8,16 +8,16 @@ jobs:
   build-and-upload-assets:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v4
       - uses: shivammathur/setup-php@v2
         with:
           php-version: 8.2
           extensions: zip
           coverage: none
           tools: phive
       - id: composer-cache
-        run: echo "::set-output name=dir::$(composer config cache-files-dir)"
-      - uses: actions/cache@v2
+        run: echo "dir=$(composer config cache-files-dir)" >> $GITHUB_OUTPUT
```

Comment:
What are you try to do?

Reply:
Fix the deprecation messages in github actions. Example: https://github.com/dbrekelmans/browser-driver-installer/actions/runs/6144208291

```The **set-output** command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/```


The other changes are
```The following actions uses node12 which is deprecated and will be forced to run on node16: actions/checkout@v2, actions/cache@v2, bruceadams/get-release@v1.2.2, actions/upload-release-asset@v1, peter-evans/repository-dispatch@v1. For more info: https://github.blog/changelog/2023-06-13-github-actions-all-actions-will-run-on-node16-instead-of-node12-by-default/```